packages/quantic/force-app/main/default/lwc/quanticPager/e2e/quantic-pager.e2e.ts

```diff
@@ -0,0 +1,229 @@
+import {Page} from '@playwright/test';
+import {testSearch, testInsight, expect} from './fixture';
+import {SearchObject} from '../../../../../../playwright/page-object/search-object';
+import {
+  useCaseTestCases,
+  useCaseEnum,
+} from '../../../../../../playwright/utils/use-case';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insight-setup-object';
+import {ConfigurationObject} from '../../../../../../playwright/page-object/configuration-object';
+
+interface PageObjects {
+  configuration: ConfigurationObject;
+  search: SearchObject;
+  insightSetup?: InsightSetupObject;
+}
+
+interface PagerOptions {
+  useCase: string;
+  numberOfPages: number;
+}
+
+const numberOfResultsPerPage = 10;
+
+async function visitPage(
+  page: Page,
+  {search, insightSetup, configuration}: PageObjects,
+  options: Partial<PagerOptions> = {}
+) {
+  const pageUrl = 's/quantic-pager';
+  await page.goto(pageUrl);
+  configuration.configure(options);
+  if (options.useCase === useCaseEnum.insight) {
+    await insightSetup?.waitForInsightInterfaceInitialization();
+    await search.interceptSearchAndLimitResultPages(5);
+    await search.performSearch();
+    await search.waitForSearchResponse();
+  } else {
+    await search.interceptSearchAndLimitResultPages(5);
+    await search.waitForSearchResponse();
+  }
+}
+
+async function visitPageAndloadOptionsFromUrl(
+  page: Page,
+  {search, configuration}: PageObjects,
+  urlHash: string
+) {
+  const pageUrl = 's/quantic-pager';
+  await page.goto(`${pageUrl}#${urlHash}`);
+  await search.interceptSearchAndLimitResultPages(5);
+  configuration.configure();
+}
+
+async function setupWithPauseBeforeSearch(
+  page: Page,
+  {search, insightSetup, configuration}: PageObjects,
+  options: Partial<PagerOptions> = {}
+) {
+  const pageUrl = 's/quantic-pager';
+  await page.goto(pageUrl);
+
+  if (options.useCase === useCaseEnum.insight) {
+    configuration.configure(options);
+    await insightSetup?.waitForInsightInterfaceInitialization();
+    const stalledRequest = search.interceptSearchIndefinitely();
+    await search.performSearch();
+    return stalledRequest;
+  }
+  const stalledRequest = search.interceptSearchIndefinitely();
+  configuration.configure(options);
+  return stalledRequest;
+}
+
+const fixtures = {
+  search: testSearch,
+  insight: testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+
+  test.describe(`quantic pager ${useCase.label}`, () => {
+    test.beforeEach(async ({page, search, insightSetup, configuration}) => {
+      await visitPage(
+        page,
+        {search, insightSetup, configuration},
+        {useCase: useCase.value}
+      );
+    });
+
+    test.describe('when the page is loading ${useCase.label', () => {
```

Comment:
```suggestion
    test.describe(`when the page is loading ${useCase.label}`, () => {
```
That probably didn't work right?